common/views/themes/config.ts

```diff
@@ -135,6 +136,24 @@ export const sizes = {
   headerLarge: 1040,
 };
 
+const defaultButtonColors: ButtonColors = {
+  text: 'white',
+  background: 'green',
+  border: 'green',
+};
+
+const dangerButtonColors: ButtonColors = {
+  text: 'white',
+  background: 'red',
+  border: 'red',
+};
+
+const needsABetterName: ButtonColors = {
```

Comment:
Any suggestions?

Reply:
I would call it `exhibitionGuideButton`; a better name will emerge when we discover where we're going to use it next.

Or `whiteBackgroundButton`

Reply:
I think [we're going to go with [border][Background][text] for now](https://wellcome.slack.com/archives/C03SU2ZNR9D/p1660230609446119), and get a UX/design conversation started around what to expect the button requirements to look like